app/models/import/builder.rb

```diff
@@ -0,0 +1,13 @@
+class Import
+  class Builder
+    def initialize(gqueries, id:, scaling:, **)
+      @gqueries    = gqueries
+      @scenario_id = id
+      @scaling     = scaling
+    end
+
+    def build(response = nil)
```

Comment:
The convention is to use an underscore before the variable name (or just an underscore on its own) *when the variable isn't used*. In fact, Ruby itself will warn you if you define a normal variable (not a method argument) which isn't used, [*unless* it starts with an underscore](https://bugs.ruby-lang.org/issues/6693).

```ruby
unused       = true
also_unused  = true
_underscored = true
```

```
ruby -w unused_variables.rb

unused_variables.rb.rb:1: warning: assigned but unused variable - unused
unused_variables.rb.rb:2: warning: assigned but unused variable - also_unused
```

[Rubocop extends this](https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/lint/unused_method_argument.rb) to also warn about  unused block and method arguments.

If you use the variable inside the method, don't add the underscore.

```ruby
class Builder
  def build(_response)
    # Doesn't use the response object
  end
end

class HouseBuilder < Builder
  def build(response)
    # Does use the response object
  end
end
```

But what I really mean is: why is it `build(response = nil)` and not just `build(response)`? When would you ever call `build` without a response (or a fake stub response in the specs)? It results in the public API being different: we can call `builder.build`, but not `house_builder.build`.